### DISCUSSION TITLE
Please allow removal of 4px padding around Select Dropdown Items

### DISCUSSION THREAD

**Discussion by ahulterstrom** [mantinedev/mantine#2216] [Feature requests]
The `SelectPopover.Dropdown` has this line which applies 4px of padding to a parent div containing all of the dropdown items:
```
<div style={{ display: 'flex', flexDirection: direction, width: '100%', padding: 4 }}>
```
Because this is set with inline styles, the `padding: 4` is difficult to override, which is a problem when trying to achieve dropdown designs which do not have padding around the items, such as this example:

![image](https://user-images.githubusercontent.com/55113250/186289800-d33db700-5c44-4a83-a7d0-2834f2e659d6.png)

Would there be any way this could be changed so that the padding can be adjusted with the styles api?

**Comment by chrisganov** [comment]
Hey @ahulterstrom  I had the same issue and this is what i found as a very hacky workaround:

1. create a forwardRef function for "dropdownComponent"
            dropdownComponent={React.forwardRef((props, ref) => (
              <Box ref={ref} {...props} className={classes.customContainer}>
            ))}

2. use the following selector in the createStyles:
  customContainer: {
    width: "100%",
    
    "& > div:nth-of-type(1)": {
      padding: "0px !important",
    },
